include/eve/arch/riscv/rvv_common_masks.hpp

```diff
@@ -0,0 +1,56 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/detail/function/friends.hpp>
+
+namespace eve::detail
+{
+template<arithmetic_scalar_value T, typename N>
+EVE_FORCEINLINE logical<wide<T, N>>
+                rvv_true()
+{
+  static constexpr auto   lmul     = riscv_rvv_dyn_::getLMUL<T>(N::value);
```

Comment:
I'm struggling with `riscv_rvv_dyn_`. I can't find where it;s defined.
Is it that different machines use different number of bits per logical element?

Like on one machine logical<char> is represented by 1 bit, on the other logical<char> - 8 bits?

Reply:
It is defined in tags and I use it as common place for different support function, like calculation of LMUL( vector register group count - the number of vector registers, that participate in this operation), as well as standard EVE functionality (e.g. `expected_cardinal`).

> different machines use different number of bits per logical element

Renamed. There I calculate `ratio=SEW/LMUl`.